chall03/mlindhol.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func error() {
+	fmt.Printf("whoops, error!\n")
+	os.Exit(1)
+}
```

Comment:
for the line 14, you should use fmt.Println() if you don't have variable and want to print a new line.

Plus, this function already exist in Go:

```go
log.Fatalln("whoops, error!")
```